### ISSUE TITLE
It is no longer possible to use a custom text on a sales line if another account exist with same name

### ISSUE THREAD

**Issue by peikba** [Microsoft/AL#2618]
I have two accounts for invoicing transport:

Sales Freight
Purchase Transport

If I want to use the Purchase Transport account for invoicing freight and change the description to Freight:
 
![image](https://user-images.githubusercontent.com/33752561/41507015-7877d51e-7229-11e8-92ee-c93e5dc5e7c6.png)

Then is automatically changes the Account No to the Sales Freight account
![image](https://user-images.githubusercontent.com/33752561/41507020-994cc8f8-7229-11e8-8abd-9a8f078424b5.png)

It must be possible to change the description without changing the account number.
This happened after upgrading to NAV 2018 CU6.

Can this please be removed?

**Comment by jglathe** [comment]
I would make it a configuration setting. The general rule should be if "No". contains a real item, the text-based search should be disabled. I can only imagine bad things to happen from the current behaviour. It should be possible to disable this application-wide as well, though.

**Comment by peikba** [comment]
I agree, It screwed two of my invoices up already because it changed account numbers without me noticing it.
So I disabled the functionality in table 37 (naughty boy :))

**Comment by jglathe** [comment]
Lets just guess, it is also there in T39 and T5902?

**Comment by peikba** [comment]
T39 yes but not T5902

**Comment by jglathe** [comment]
How consistent ;)

**Comment by peikba** [comment]
When was the last time changes were made to the Service module? :)

**Comment by jglathe** [comment]
T5902: 2018 CU 06, refactored line discount % calculation ;)

**Comment by hhfiddelke** [comment]
An answer would be this feature- request: [](https://github.com/Microsoft/AL/issues/2357)
Implement search as OnAfterInput- Trigger of the "No."- Field as we could do it in former NAV- Classic Client.

**Comment by thpeder** [comment]
The team has added this to their backlog and will get back to it later.
Thanks for providing feedback.
